### ISSUE TITLE
Implement something like CommNetVisualisation into RealAntennas

### ISSUE THREAD

**Issue by Gordon-Dry** [DRVeyl/RealAntennas#6]
https://github.com/KSP-TaxiService/CommNetVisualisation/tree/master/src/CommNetVisualisation/CommNetLayer

This mod is helpful in visualization of vessel comm link lines.
But it only works with vessels that got ModuleDataTransmitter.

This feature could ofc also be integral part of RO itself to be compatible with all 3 possible modules:
- ModuleDataTransmitter
- ModuleConfigurableAntenna
- ModuleRealAntenna
But I guess this will not happen.

**Comment by DRVeyl** [comment]
I agree there is a useful map view mode to show the path home for all vessels, instead of just the path for the current vessel, or all possible links.  The team with RemoteTech has a lot more experience with the UI aspects than I do.

The technical issue has nothing to do with the type of transmitter module.  The techniques shown here will work fine with RealAntennas, as well as with RO's ModuleConfigurableAntenna.  It should work fine out of the box with RO's NewComms, test it out.  It won't without changes with RA because both scenarios are fighting for control over CommNet.  (Take a look at CNVCommNetScenario, and note that RA does the same thing.)  Unfortunately, CommNet internals are not easily modified, and multiple mods trying to play in that space without collaboration is significantly more difficult.
